src/main/groovy/org/pih/warehouse/inventory/InventoryCommand.groovy

```diff
@@ -37,7 +37,7 @@ class InventoryCommand implements Validateable {
     // indicates whether non-inventory products for the warehouse should be included
     def showNonInventoryProducts = Boolean.FALSE
     // indicates whether out of stock products for the warehouse should be included
-    def showOutOfStockProducts = Boolean.FALSE
+    Boolean showOutOfStockProducts = Boolean.FALSE
```

Comment:
I think this should be also done for the booleans above since it will be causing the same issue.